client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -166,6 +166,20 @@ UndocumentedSite.prototype.removeEmailFollower = function( followerId, callback
 	}, callback );
 };
 
+UndocumentedSite.prototype.setHeaderImage = function( imageData, callback ) {
+	this.wpcom.req.post( {
```

Comment:
Excellent idea. I also recently learned that there may be another endpoint to set header images which I may use in place of this if I can find it...